libs/ngx-localstorage/src/lib/services/promisable.service.ts

```diff
@@ -16,6 +16,10 @@ export class PromisableService {
     private readonly configuration: NgxLocalstorageConfiguration,
     private readonly defaultSerializer: StorageSerializer
   ) {
+    if(this.configuration.delimiter == null || this.configuration.delimiter != null && this.configuration.delimiter == '')
```

Comment:
WHen using the default value provided by `defaultConfig` there's no check or reassignment needed here

Reply:
Fixed